src/main/java/qna/domain/Answer.java

```diff
@@ -18,9 +20,13 @@ public class Answer extends BaseEntity {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    private Long writerId;
+    @ManyToOne
+    @JoinColumn(name = "writer_id")
```

Comment:
```suggestion
    @JoinColumn
```

name이 없을 경우 어떻게 동작할지 학습해보아요.
또한 FK의 이름이 `fk_answer_writer`로 생성이 될지도 학습해보아요.

Reply:
@JoinColumn의 경우 name 속성이 없다면 기본적으로 해당 어노테이션이 붙은 `필드명_id`의 이름으로 외래키를 담는 컬럼명이 지정되네요!!

그리고 FK의 경우 아래와 같이 foreignKey의 속성을 통해 기존에 난수로 만들어지던 외래키 제약조건이 아래와 같이 제약조건을 설정할 수 있다는 점도 학습했습니다!
```java
    @ManyToOne
    @JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_writer"))
    private User writer;
```
<img width="371" alt="image" src="https://user-images.githubusercontent.com/57744251/181879354-3c713ff2-669e-45c0-8871-dc6001822551.png">